index.html

```diff
@@ -1 +1,55 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
+
+<p>The Email Markup Consortium (EMC) is a community-led group of industry professionals working to improve the email experience for everyone. Recipients should be able to easily access the content of every email sent to them, regardless of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation and sending tools, and email clients to drive this vision forward at every step of the email journey.
+
+<p>Senders, recipients, and email clients share the common goals of improving user experience, accessibility, performance, consistency, and reliability. However, currently there is a lack of cooperation in trying to achieve these goals leading to inconsistent results. By working together on agreed approaches we can save everyone time, effort, and money while delivering better results to the recipients.</p>
```

Comment:
Suggestion: Make list content from this paragraph

__Senders, recipients, and email clients share the following common goals for improving email markup:_
 
_1. user experience_
_2. accessibility_
_3. performance_
_4. consistency_
_5. reliability_ 

_Currently there is a lack of cooperation in trying to achieve these goals leading to inconsistent results.__